drift/lib/src/runtime/query_builder/expressions/null_check.dart

```diff
@@ -29,6 +29,22 @@ Expression<T> coalesce<T extends Object>(List<Expression<T>> expressions) {
   return FunctionCallExpression<T>('COALESCE', expressions);
 }
 
+/// The ifnull() function returns a copy of its first non-NULL argument, or
+/// NULL if both arguments are NULL
+Expression<T> ifnull<T extends Object>(Expression<T> x, Expression<T> y) {
+  return FunctionCallExpression<T>('IFNULL', [x, y]);
+}
+
+/// The nullif(X,Y) function returns its first argument if the arguments are
+/// different and NULL if the arguments are the same. The nullif(X,Y) function
+/// searches its arguments from left to right for an argument that defines a
+/// collating function and uses that collating function for all string
+/// comparisons. If neither argument to nullif() defines a collating function
+/// then the BINARY collating function is used.
+Expression<T> nullif<T extends Object>(Expression<T> x, Expression<T> y) {
```

Comment:
I think this could also be a method / an extension on `Expression<T>`, so that you'd be able to write `x.nullIf(y)`.